packages/react-slider/src/components/Slider/stories/SliderControlled.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import { Label } from '@fluentui/react-label';
```

Comment:
```suggestion
import { Label } from '@fluentui/react-label';
```

you still need ts-ignore unfortunately until #20597 is completed

Reply:
I think the ts-ignore can be removed now?